src/main/java/com/fabriccommunity/spookytime/MixinHelpers.java

```diff
@@ -0,0 +1,26 @@
+package com.fabriccommunity.spookytime;
+
+import net.minecraft.entity.EquipmentSlot;
+import net.minecraft.entity.LivingEntity;
+import net.minecraft.item.ItemStack;
+import net.minecraft.item.Items;
+
+import java.util.Random;
+
+/**
+ * @author Indigo Amann
+ */
+public class MixinHelpers {
+    public static final Random random = new Random();
+    public static ItemStack getEquippedPumpkin(LivingEntity entity, EquipmentSlot slot) {
+        if (slot != EquipmentSlot.HEAD) return entity.getEquippedStack(slot);
```

Comment:
I would prefer to have braces here, just for extra clarity.

Reply:
Either braces or ternary.